# This makes sure that the following test works when the closed-over var is not
# within __main__ as a global.
import close_loop_var_in_func
